### PR TITLE
fix: Employee Onboarding/Seperation task Assignment

### PR DESCRIPTION
**Pull request by Anurag810**
Please read the pull request checklist to make sure your changes are merged: https://github.com/frappe/erpnext/wiki/Pull-Request-Checklist

### CODE REVIEW

File: erpnext/hr/utils.py [hunk 1]
```diff
@@ -54,10 +54,17 @@ def on_submit(self):
 					where parenttype='User' and role=%s''', activity.role)
 				users = users + user_list
 
+				if "Administrator" in users:
+					users.remove("Administrator")
+
+
 			# assign the task the users
 			if users:
+				print(users)
```
Comment (sagarvora) [inline erpnext/hr/utils.py]: please remove print statement
```suggestion
```

Comment (Anurag810) [reply erpnext/hr/utils.py]: oh removed

File: erpnext/hr/utils.py [hunk 2]
```diff
@@ -54,10 +54,16 @@ def on_submit(self):
...
 
+				if "Administrator" in users:
+					users.remove("Administrator")
+
+
 			# assign the task the users
 			if users:
 				self.assign_task_to_users(task, set(users))
 
+			users = []
```
Comment (sagarvora) [inline erpnext/hr/utils.py]: what's the use for emptying the array?

Comment (sagarvora) [reply erpnext/hr/utils.py]: fixed for now.

**Comment by sagarvora** [comment]
for semantic PRs, please ensure that the PR title is written in lowercase (`fix` instead of `Fix`)
also removed unnecessary code for you.

**Review by sagarvora — APPROVED** [review]

**Pull request merged**